src/github/pagination.ts

```diff
@@ -1,28 +0,0 @@
-import { AnyResponse } from '@octokit/rest'
-import { GitHubAPI } from './'
-
-export function addPagination (octokit: GitHubAPI) {
-  octokit.paginate = paginate.bind(null, octokit)
-}
-
-const defaultCallback = (response: Promise<AnyResponse>, done?: () => void) => response
-
-async function paginate (octokit: GitHubAPI, responsePromise: any, callback = defaultCallback) {
-  let collection: any[] = []
-  let getNextPage = true
-
-  const done = () => {
-    getNextPage = false
-  }
-
-  let response = await responsePromise
-
-  collection = collection.concat(await callback(response, done))
-
-  // eslint-disable-next-line no-unmodified-loop-condition
-  while (getNextPage && octokit.hasNextPage(response)) {
-    response = await octokit.getNextPage(response)
-    collection = collection.concat(await callback(response, done))
-  }
-  return collection
-}
```

Comment:
@tcbyrd note that Probot’s current pagination method is different from the one of `@octokit/rest`, we need to mention that in the release notes. We could also keep both implementations and then switch based on input. If the user passed in a promise (Probot’s implementation) we could log a deprecation warning and then remove it with the next breaking release